interface-definitions/high-availability.xml.in

```diff
@@ -252,6 +252,144 @@
+                <description>Source hashing</description>
+              </valueHelp>
+              <valueHelp>
+                <format>destination-hashing</format>
+                <description>Destination hashing</description>
+              </valueHelp>
+              <valueHelp>
+                <format>locality-based-least-connection</format>
+                <description>Locality-Based least connection</description>
+              </valueHelp>
+              <constraint>
+                <regex>^(round-robin|weighted-round-robin|least-connection|weighted-least-connection|source-hashing|destination-hashing|locality-based-least-connection)$</regex>
+              </constraint>
+            </properties>
+            <defaultValue>least-connection</defaultValue>
+          </leafNode>
+          <leafNode name="delay-loop">
+            <properties>
+              <help>Interval between health-checks (in seconds)</help>
+              <valueHelp>
+                <format>u32:1-600</format>
+                <description>Interval in seconds (default: 10)</description>
+              </valueHelp>
+              <constraint>
+                <validator name="numeric" argument="--range 1-3600"/>
+              </constraint>
+            </properties>
+            <defaultValue>10</defaultValue>
+          </leafNode>
+          <leafNode name="forward-method">
+            <properties>
+              <help>Forwarding method (default - NAT)</help>
+              <completionHelp>
+                <list>direct nat tunnel</list>
+              </completionHelp>
+              <valueHelp>
+                <format>direct</format>
+                <description>Direct routing</description>
+              </valueHelp>
+              <valueHelp>
+                <format>nat</format>
+                <description>NAT</description>
+              </valueHelp>
+              <valueHelp>
+                <format>tunnel</format>
+                <description>Tunneling</description>
+              </valueHelp>
+              <constraint>
+                <regex>^(direct|nat|tunnel)$</regex>
+              </constraint>
+            </properties>
+            <defaultValue>nat</defaultValue>
+          </leafNode>
+          #include <include/port-number.xml.i>
+          <leafNode name="persistence-timeout">
+            <properties>
+              <help>Timeout for persistent connections</help>
+              <valueHelp>
+                <format>u32:1-86400</format>
+                <description>Timeout for persistent connections (default 300)</description>
+              </valueHelp>
+              <constraint>
+                <validator name="numeric" argument="--range 1-86400"/>
+              </constraint>
+            </properties>
+            <defaultValue>300</defaultValue>
+          </leafNode>
+          <leafNode name="protocol">
+            <properties>
+              <help>Protocol for port checks (default TCP)</help>
```

Comment:
Another case of inconsistent default value help formatting.

Reply:
Done